mslib/utils/qt.py

```diff
@@ -446,13 +446,15 @@ def __init__(self, parent=None):
         self.is_git_env = False
         self.new_version = None
         self.old_version = None
-        self.command = "conda"
-
+        # we are using the installer version of the env
+        self.conda_prefix = os.getenv("CONDA_PREFIX")
+        self.command = os.path.join(self.conda_prefix, 'bin', "conda")
```

Comment:
There is a naked "conda" below in line ~490

Reply:
ouch, that has to become self.command